core/master/src/main/java/org/drftpd/commands/usermanagement/UserManagementHandler.java

```diff
@@ -1817,12 +1817,27 @@ public CommandResponse doSITE_USER(CommandRequest request) throws ImproperUsageE
 
         Session session = request.getSession();
 
-        if (request.getUser() == null) {
```

Comment:
Should not be removed. If the requestedUser is null, the command must be aborted. ACCESS DENIED is send but i think to not possibly disclose that the user exists.

Reply:
You are correct. Fixing